assets/l10n/app_hi.arb

```
@@ -329,7 +329,9 @@
   "enter_additional_info_req_id": "अतिरिक्त जानकारी अनुरोध आईडी दर्ज करें",
   "additional_info_req_id": "अतिरिक्त जानकारी अनुरोध आईडी",
   "no_access_to_this_page": "आपको इस पृष्ठ तक पहुँच नहीं है",
+  "server_unreachable": "प्रमाणीकरण सर्वर उपलब्ध नहीं है। कृपया बाद में पुनः प्रयास करें",
   "scheduled_job_settings": "अनुसूचित कार्य सेटिंग्स",
+  "server_unreachable": "प्रमाणीकरण सर्वर उपलब्ध नहीं है। कृपया बाद में पुनः प्रयास करें",
```

RuboCop (1.81.7)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)